sdk/python/feast/pyspark/launchers/standalone/local.py

```diff
@@ -230,7 +293,7 @@ def historical_feature_retrieval(
             self.spark_submit(job_params),
             job_params.get_destination_path(),
         )
-        JOB_CACHE[job_id] = job
+        JOB_CACHE.add_job(job_id, None, job)
```

Comment:
By the way, why is `JOB_CACHE` all caps?

Reply:
It's a global constant variable (in a sense that we assign value to it once and never change the reference). Do you have a different naming convention for such variables?